content/en/logs/guide/delete_logs_with_sensitive_data.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Delete Logs with Sensitive Data
+kind: guide
+disable_toc: false
+further_reading:
+- link: "logs/guide/control-sensitive-logs-data/"
+  tag: "Documentation"
+  text: "Control Sensitive Logs Data"
+- link: "/sensitive_data_scanner/"
+  tag: "Documentation"
+  text: "Sensitive Data Scanner"
+---
+
+## Overview
+
+It is important to delete logs with sensitive data to ensure the security of your data. This guide provides information on how to:
+
+- Check if the logs with sensitive data need to be deleted because they are within the retention period.
+- Make logs with sensitive data un-queryable.
+- Redact sensitive data with Sensitive Data Scanner.
+- Request log deletion from Datadog support.
+
+## Check your log retention period
+
+Datadog automatically deletes logs when the longest retention period configured for your organization has been met. Therefore, any logs that have passed the longest retention period are deleted automatically. Do the following to check your log retention period and how you can change the retention period:
+
+1. Navigate to the [Log Indexes][1] page.
+1. See the log retention period for each index in the **Retention** column.
+1. If you want to change the retention period of an index so that the logs age out faster, click the edit icon on the right side of the index.
+1. Update the **Set Index Retention** drop down menu to a new retention period.
+
+## Make logs with sensitive data un-queryable
+
+If logs with sensitive data are within the the log retention period, you can make then un-queryable in Datadog's Log Explorer, Dashboards, and Live Tail until they age out. Logs made un-queryable are not available for querying or viewing. Follow these [instructions][2] to make logs with sensitive data un-queryable in Datadog.
+
+## Deleting an entire index
+
+Do the following if you need to delete an entire index:
+
+1. Navigate to the [Log Indexes][1] page.
+1. Click on the delete icon on the right side of the index you want to delete.
+1. Click **Confirm** to delete the index.
+
+**Note**: The index shows as a pending deletion until the logs age out, after which the index is fully deleted and removed from the UI.
+
+## Redact sensitive data with Sensitive Data Scanner
+
+Use [Sensitive Data Scanner][5] to limit the risk of storing sensitive data in Datadog. Sensitive Data Scanner is a stream-based, pattern matching service used to identify, tag, and optionally redact or hash sensitive data. Security and compliance teams can implement Sensitive Data Scanner to prevent sensitive data leaks and limit non-compliance risks.
+
+## Submit a request for log deletion
+
+<div class="alert alert-warning">
+Only a Datadog Admin can request log deletion. If you are not an Admin, make sure to include an Admin to confirm the deletion request.
```

Comment:
Just wanted to call out I'm not sure what is meant exactly by "include an Admin" here. Is this something you do in the UI? If this is something a user would know, please ignore my comment. 😄

Reply:
@brett0000FF good point! It's about looping in an Admin on the request. What do you think about:

> Only a Datadog Admin can request log deletion. If you are not an Admin, make sure to include an Admin on the request so they can confirm the deletion request.

Reply:
Perfect, that's clearer to me!